packages/adblocker/src/filters/network.ts

```diff
@@ -400,6 +396,11 @@ function compileRegex(
   return new RegExp(filter);
 }
 
+/**
+ * Finds the last index of an unescaped character in the given string.
+ * This function tries to find the match from the backward.
+ * When this function sees an escaping character, it will jump to the next index.
+ */
 export function findLastIndexOfUnescapedCharacter(text: string, character: string) {
```

Comment:
Isn't this function in utils already? (same for the two functions below)

Reply:
I think these are duplicated while rebasing, fixed and removed obsoletes.